.github/workflows/stale.yml

```diff
@@ -0,0 +1,19 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "30 1 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
```

Comment:
Looks like v3 is the latest major stable release.
https://github.com/actions/stale/releases

Reply:
```suggestion
    - uses: actions/stale@v3
```